website/content/en/preview/concepts/disruption.md

```diff
@@ -177,6 +181,63 @@ To enable interruption handling, configure the `--interruption-queue-name` CLI a
 
 ## Controls
 
+### Disruption Budgets
+
+You can rate limit Karpenter's disruption through the `NodePool.Spec.Disruption.Budgets`. If undefined, Karpenter will default to one budget with `nodes: 10%`. Budgets will take into account nodes that are being deleted for any reason, and will only block Karpenter from disrupting nodes voluntarily through expiration, drift, emptiness, and consolidation.
+
+#### budget.Duration + budget.Schedule
+Duration and Schedule can only be defined together. When omitted, the budget is always active. When defined, the schedule determines when a budget is active, and the duration determines how long from that point the budget is active. 
+
+##### Schedule
+Schedule is a cronjob schedule. Generally, the cron syntax is five space-delimited values with options below. 
```

Comment:
Should we use the word "generally" here? This is always true, right? Or am I missing a different way that crons can be written out?

Reply:
Nope, you can use the @yearly or @daily https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#:~:text=Other%20than%20the%20standard%20syntax%2C%20some%20macros%20like%20%40monthly%20can%20also%20be%20used%3A

Reply:
Is it worth just calling out the different ways if there is a minimal set? Rather than saying something like "most of the time" and then not offering the alternatives. If the alternative set is too large, that's fine, just seems like the statement is asking for someone asking, "but what about the non-general case?"

Reply:
I was thinking the link in the markdown doc has this information, so I didn't want to inflate the doc too much with essentially duplicated information. I added the small list and modified the text here.